mlir/lib/Driver/CompilerDriver.cpp

```diff
@@ -742,32 +756,32 @@ LogicalResult QuantumDriverMain(const CompilerOptions &options, CompilerOutput &
         llvmModule->setDataLayout(targetMachine->createDataLayout());
         llvmModule->setTargetTriple(targetTriple);
 
-        catalyst::utils::LinesCount::Module(*llvmModule.get());
-
-        TimingScope coroLLVMPassesTiming = llcTiming.nest("LLVM coroutine passes");
-        if (options.asyncQnodes &&
-            failed(timer::timer(runCoroLLVMPasses, "runCoroLLVMPasses", /* add_endl */ false,
-                                options, llvmModule, output))) {
-            return failure();
+        if (options.asyncQnodes) {
+            TimingScope coroLLVMPassesTiming = llcTiming.nest("LLVM coroutine passes");
+            if (failed(timer::timer(runCoroLLVMPasses, "runCoroLLVMPasses", /* add_endl */ false,
```

Comment:
Are these two different timing systems?

Reply:
Yes the timer::timer is for the frontend instrumentation, and the TimingScope one is for mlir-opt internal timing which enables us to use --mlir-timing option with catalyst-cli which would print a similar time breakdown.

Reply:
Oh neat, I'll have to try it at some point!